gatsby-config.js

```diff
@@ -89,6 +89,14 @@ module.exports = {
         path: `${__dirname}/src/content/pages`,
       },
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `refugee-guide`,
+        path: `${__dirname}/src/content/refugee-guide`,
+      },
+    },
+    `gatsby-transformer-remark`,
```

Comment:
This is already defined on [line 107](https://github.com/adaptdk/su-ukraina/pull/37/files#diff-b5e305780d9d473da97c61beab8bc36e5e8871b360942e4686c9b20d8c5d4cfaR107)